src/components/Card/Card.jsx

```diff
@@ -0,0 +1,43 @@
+import { FiMinus } from "react-icons/fi";
+import PropTypes from "prop-types";
+
+import ReadingTime from "./ReadingTime";
+import Button from "../shared/Button/Button";
+
+function Card({ favicon, domain, articleTitle, readingTime }) {
+  return (
+    <li className="relative content-center w-40 h-40 pl-5 list-none transition-all delay-75 bg-white shadow-md group hover:scale-125 shadow-black/25 rounded-3xl">
+      <a
+        href="https://www.google.com"
+        target="_blank"
+        className="relative -top-1"
+        rel="noreferrer"
+      >
+        <div>
+          <img
+            className="inline-block w-4"
+            src={favicon || "public/favicon.png"}
+            alt="favicon"
+          />
+          <p className="text-[11px] inline-block ml-1 font-medium">{domain}</p>
+        </div>
+        <ReadingTime readingTime={readingTime} />
+        <p className="mt-1 mr-5 group-hover:pointed-title font-extralight title-ellipsis">
+          {articleTitle}
+        </p>
+      </a>
+      <Button>
+        <FiMinus color="gray" strokeWidth={4} />
+      </Button>
+    </li>
+  );
+}
+
+export default Card;
+
+Card.propTypes = {
+  favicon: PropTypes.string.isRequired,
```

Comment:
favicon이 없는 경우가 있을 것 같은데 isRequired 가 있어도 되는지 한 번 더 생각해 봐야 할 듯합니다!
```suggestion
favicon: PropTypes.string,
```